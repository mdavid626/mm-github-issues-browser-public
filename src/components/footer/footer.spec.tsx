import { cleanup, render } from '@testing-library/react';
import React from 'react';
import Footer from './footer';

describe('footer', () => {
  afterEach(cleanup);
  afterEach(jest.resetAllMocks);

  it('should render', () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
