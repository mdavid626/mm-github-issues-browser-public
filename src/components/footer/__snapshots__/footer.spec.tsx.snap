// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer should render 1`] = `
<DocumentFragment>
  <div
    class="Footer"
  >
    Dávid Molnár © 2023
  </div>
</DocumentFragment>
`;
